cdisc-org/cdisc-rules-engine
# Import models that are imported inline by odmlib to support compiled …

…define validation

Steps to test

1. Compile core using the pyinstaller command found in the README
2. Run a validation with directory including a define 2.1 file: ex `./dist/core/core validate -s sdtmig -v 3-4 -d msg/`
3. Verify the validation runs completely and an excel file is generated
